boefjes/boefjes/plugins/kat_rpki/normalize.py

```diff
@@ -10,14 +11,18 @@ def run(input_ooi: dict, raw: bytes) -> Iterable[NormalizerOutput]:
     results = json.loads(raw)
     ooi = Reference.from_str(input_ooi["primary_key"])
 
-    if not results["exists"]:
-        ft = KATFindingType(id="KAT-NO-RPKI")
-        f = Finding(finding_type=ft.reference, ooi=ooi)
-        yield ft
-        yield f
+    address = IPv4Address(ooi.tokenized.address)
```

Comment:
Does not seem to support IPv6? ipaddress.ip_address is the dual stack supporting function for this.

Reply:
Thanks! Stupid.... :)